packages/navie/src/prompt.ts

```diff
@@ -129,6 +131,12 @@ information about installing, configuring, and using AppMap.`,
     prefix: 'Help document',
     multiple: true,
   },
+  [PromptType.CodeEditor]: {
+    content: `**Code editor**
+
+You're provided with information about the user's code editor. This information includes the code editor's name.`,
```

Comment:
I don't think this is a separate prompt, but rather it's part of the ProjectInfo.